src/app/ReadClient.cpp

```diff
@@ -403,6 +403,7 @@ CHIP_ERROR ReadClient::BuildDataVersionFilterList(DataVersionFilterIBs::Builder
                                                   const Span<DataVersionFilter> & aDataVersionFilters,
                                                   bool & aEncodedDataVersionList)
 {
+    ChipLogProgress(DataManagement, "Attempting to encode %lu data version filters", static_cast<unsigned long>(aDataVersionFilters.size()));
```

Comment:
We could move this to the end so in either case we get one log statement saying how many we encoded

Reply:
The "we didn't encode everything" log statement logs how many were chopped off.  Not that this is not trivially derivable from "how many we encoded" and "how many we were asked to encode", since we skip encoding the ones that don't match our request paths.....

Reply:
Changed to one log statement.